source/styles.brs

```diff
@@ -1,78 +1,17 @@
-function GetStyles() as Object
-    return {
-        classes: {
-            ' HomeScene
-            homeList: {
-                itemComponentName: "HomeListItem",
-                numRows:"3",
-                translation: "[50, 150]",
-                itemSize: "[ 2000, 350 ]",
-                showRowLabel: "[ true ]",
-                rowLabelOffset: "[ [0, 15], [0, 15], [0, 15] ]",
-                drawFocusFeedback:true,
-                vertFocusAnimationStyle:"fixedFocusWrap",
-                rowFocusAnimationStyle: "fixedFocusWrap",
-                visible: true,
-                rowItemSize:"[[500, 285]]",
-                itemSpacing:"[0, 80]",
-                rowItemSpacing: " [[30, 0]]"
-            },
-            video: {
-                   width:"1920",
-                   height:"1080",
-                   visible:false
-            },
+function getClassesFromJson () as object
+      info = ReadAsciiFile("pkg:/source/styles.json")
+      json = ParseJson(info)
 
-            videoDetail:{
-                visible: false
-            },
+      'contentNode = CreateObject("roSGNode", "ContentNode")
 
-            'Home List item
-            homeListItemLayout: {
-                  layoutDirection: "vert",
-                  vertAlignment:"top",
-                  translation:"[0, 0]"
-            },
+      classes= CreateObject("roAssociativeArray")
 
-            homeListItemPoster: {
-                  width:"500",
-                  height: "290"
-            },
+      for each selector in json
+          style = CreateObject("roAssociativeArray")
+          style = json[selector]
+          ' for each item in json[selector]
+          classes[selector] = style
 
```

Comment:
why are you doing this?
what's the difference between `classes` and `json`?

Reply:
Sorry, lack of creativity with the variable names. :P